v2/periphery/LockupLinearBatchStreamCreator.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity >=0.8.13;
+
+import { IAllowanceTransfer } from "@permit2/interfaces/IAllowanceTransfer.sol";
```

Comment:
We should re-export this interface in `@sablier/v2-periphery`, in the `src/types` directory.

Reply:
We could

Reply:
https://github.com/sablier-labs/v2-periphery/issues/130